src/Kmd.Logic.Cpr.Client/CprClient.cs

```diff
@@ -209,7 +209,7 @@ public async Task<bool> UnsubscribeByIdAsync(Guid id)
         /// <param name="pageNo">The page number to query, starting at 1.</param>
         /// <param name="pageSize">The maximum number of results to return.</param>
         /// <returns>List of citizen records.</returns>
-        public async Task<object> GetAllCprEvents(DateTime dateFom, DateTime dateTo, int pageNo, int pageSize)
+        public async Task<object> GetAllCprEventsAsync(DateTime dateFom, DateTime dateTo, int pageNo, int pageSize)
```

Comment:
Can we please not return an object. We should do a better job than autorest does and return the structured type. That often means checking the HttpSatusCode and throwing an exception if required

Reply:
Fixed